ani-cli

```diff
@@ -1,25 +1,37 @@
 #!/bin/sh
 
-# version number
-VERSION="1.6.5"
+# ani-cli
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Project repository: https://github.com/pystardust/ani-cli
+# Version number:
+VERSION='1.6.5'
 
 
-# history file path
-logfile="${XDG_CACHE_HOME:-$HOME/.cache}/ani-hsts"
-auto_play=0
-
 #######################
-# Auxiliary functions #
+# AUXILIARY FUNCTIONS #
 #######################
 
 help_text () {
 	while IFS= read -r line; do
-		printf "%s\n" "$line"
+		echo "$line"
```

Comment:
why are you using `echo` here? 
Read this https://mywiki.wooledge.org/BashPitfalls#echo_.24foo
Avoid using echo cause the posix implementation of it is questionable. many shells have different implementations of echo even tho they are posix complaint.

Reply:
basically
```
echo "hi\nthere"
```
word differently in dash and bash

Reply:
I thought it was more readable, I wasn't aware how skimpy it is.